README.md

```diff
@@ -22,6 +22,18 @@ Knot.x Fragments is designed to build fault-tolerant, back-end integrations such
 - API Gateways, Backend For Frontend (BFF) for single-page applications (SPA), Web APIs
 - documents processing (HTML, JSON, PDF etc) with a templating engine support
 
+## Modules
+
+- [Fragments Supplier](https://github.com/Knotx/knotx-fragments/tree/master/supplier) - converts a HTTP request into one or more [**Fragments**](https://github.com/Knotx/knotx-fragments/tree/master/api#knotx-fragment-api)
```

Comment:
Don't know if it's correct, but somehow I feel that `HTTP` should be preceeded with `an`. Call me purist 😆